### DISCUSSION TITLE
Server build

### DISCUSSION THREAD

**Discussion by Polygons1** [leptos-rs/leptos#2617] [Q&A] [answered]
How do I build my app server for wasm32-wasi or is it impossible?

**Comment by gbj** [comment]
Not sure which platform you're looking at running it on but you can look at [`leptos-spin`](https://github.com/fermyon/leptos-spin) for example.

**Reply by Polygons1** [reply]
trying to run the server on x86_64 Linux with Wasmer and I also saw that there is a server_is_wasm option in the cargo-leptos code and I just don’t know what to put in my config file for it to be true

**Reply by gbj** [reply]
I don't know anything about Wasmer, sorry. Feel free to share an example and what errors or issues are arising.

**Reply by Polygons1** [reply]
first I need to compile, I saw this: https://github.com/leptos-rs/cargo-leptos/blob/159f6494ca5b27a3e659b8f79e1f31512dd3b1b6/src/compile/server.rs#L78 and I want to get it to be true but I don't know what is the config for it

**Reply by gbj** [reply, marked as answer]
That's a variable. In the line above you see it's `true` if the bin target triple includes `wasm`. You can set that with the `bin-target-triple` config variable (see cargo-leptos README)
